examples/header-group.html

```diff
@@ -0,0 +1,80 @@
+<template id=header-group>
+
+<style>
+
+  header-group
+    { display: block }
+
+  header-group
+    hgroup
+      { background: #bada55; color: var(--darkest-color) }
+
+</style>
+
+<hgroup>
+  <h1><a href=/examples/index.html>snuggsi ツ</a></h1>
+  <slot name=header><h2>try an example</h2></slot>
+</hgroup>
+
+</template>
+
+<script>
+const tag = `header-group`
+
+Element `header-group`
+
+(class extends HTMLElement {
+
+  static onimportload (event) {
```

Comment:
@brandondees we should probably upstream this entire event handler to `GlobalEevntHandler` interface mixin. This way will be available to all custom elements if a respective `link rel=import`

https://github.com/devpunks/snuggsi/blob/master/elements/global-event-handlers.es

`GlobalEventHandler.onload` (seems spec may be out of date as well since no mention of `<link rel=import onload>` event. https://developer.mozilla.org/en-US/docs/Web/API/GlobalEventHandlers/onload